src/components/ui/widget.tsx

```diff
@@ -2,10 +2,15 @@ import { ReactNode } from 'react';
 
 interface WidgetProps {
   children: ReactNode;
+  className?: string;
 }
 
-const Widget = ({ children }: WidgetProps) => {
-  return <div className='w-[576px] rounded-xl bg-secondary'>{children}</div>;
+const Widget = ({ children, className }: WidgetProps) => {
+  return (
+    <div className={'w-[576px] rounded-xl bg-secondary px-6 py-1 ' + className}>
```

Comment:
Use the helper function `cn()` to concat tailwind classes.

1. Import `cn` from `@/lib/utils`
2. Use:
 ```html
<div {className={cn('w-[576px] rounded-xl bg-secondary px-6 py-1', className)} />
 ```

This combine all possible classes who are passed in props and merge it without conflits.

Reply:
fixed at [this commit](https://github.com/Super-Patos-Hackaton/frontend/commit/85bc4f994d3b198346a0fcc182cb62c1c175d81c)